base/array.jl

```diff
@@ -530,6 +530,7 @@ oneunit(x::AbstractMatrix{T}) where {T} = _one(oneunit(T), x)
 ## Conversions ##
 
 convert(::Type{T}, a::AbstractArray) where {T<:Array} = a isa T ? a : T(a)
+convert(::Type{Union{}}, a::AbstractArray) = throw(MethodError(convert, (Union{}, a)))
```

Comment:
This comes from a `convert(Nothing, ::Any)`. Will that be handled by your other PR?

Reply:
I think this is a separate invalidation coming from `Union{}(::Vector{Any})`, because StaticArrays defines `convert(::Type{<:StaticArray}, ::AbstractArray)`. Or did I misunderstand you? (The invalidations above are with #39434 already)